nemoguardrails/rails/llm/llmrails.py

```diff
@@ -377,10 +374,9 @@ def _prepare_model_kwargs(self, model_config):
             if api_key:
                 kwargs["api_key"] = api_key
 
-        # enable streaming token usage when streaming is enabled
+        # enable streaming token usage
         # providers that don't support this parameter will simply ignore it
-        if self.config.streaming:
-            kwargs["stream_usage"] = True
+        kwargs["stream_usage"] = True
```

Comment:
@Pouyanpi : we can move this later to llm_call when we drop the streaming callback handler feature.